sdk/elixir/runtime/main.go

```diff
@@ -19,35 +19,9 @@ const (
 
 func New(
 	// +optional
+	// +defaultPath="/sdk/elixir"
 	sdkSourceDir *dagger.Directory,
 ) *ElixirSdk {
-	if sdkSourceDir == nil {
-		// TODO: Replace with a *default path from context* when
-		// https://github.com/dagger/dagger/pull/7744 becomes available.
-		sdkSourceDir = dag.Directory().
-			// NB: these patterns should match those in `dagger.json`.
-			// When `--sdk` points to a git remote the files aren't filtered
-			// using `dagger.json` include/exclude patterns since the whole
-			// repo is cloned. It's still useful to have the same patterns in
-			// `dagger.json` though, to avoid the unnecessary uploads when
-			// loading the SDK from a local path.
-			WithDirectory(
-				"/",
-				dag.CurrentModule().Source().Directory(".."),
-				dagger.DirectoryWithDirectoryOpts{
-					Include: []string{
-						"LICENSE",
-						"lib/**/*.ex",
-						".formatter.exs",
-						"mix.exs",
-						"mix.lock",
-						"dagger_codegen/lib/**/*.ex",
-						"dagger_codegen/mix.exs",
-						"dagger_codegen/mix.lock",
-					},
```

Comment:
As I understand, we still need to keep this to ensure it copies necessary files because the context directory just points to git remote files and doesn't filter any files defined in `dagger.json`.

Reply:
Re-add this statement, now it's looks the same as defined in dagger.json

```
wingyplus@WINGYMOMO ~/s/g/d/d/s/elixir (elixir-directory-context)> cat dagger.json
{
  "name": "elixir-sdk",
  "sdk": "go",
  "include": [
    "LICENSE",
    "lib/**/*.ex",
    ".formatter.exs",
    "mix.exs",
    "mix.lock",
    "dagger_codegen/lib/**/*.ex",
    "dagger_codegen/mix.exs",
    "dagger_codegen/mix.lock"
  ],
  "source": "runtime",
  "engineVersion": "v0.12.3"
}
wingyplus@WINGYMOMO ~/s/g/d/d/s/elixir (elixir-directory-context)> dagger call -m github.com/wingyplus/dagger/sdk/elixir@e8cb3a8324f908e28580d7164c7199287dde689e sdk-source-dir
Full trace at https://dagger.cloud/dagger-elixir-sdk/traces/83fd265c29a27bb3f32a7e2237408597

✔ connect 1.4s
✔ initialize 4.8s
✔ prepare 0.0s
✔ elixirSdk: ElixirSdk! 0.6s
✔ ElixirSdk.sdkSourceDir: Directory! 0.0s
✔ Directory.entries: [String!]! 0.0s

_type: Directory
entries:
    - .formatter.exs
    - LICENSE
    - dagger_codegen
    - lib
    - mix.exs
    - mix.lock
 ```